compiler/testSuite/semtype/vararg-tv.bal

```diff
@@ -0,0 +1,24 @@
+//@type FT1 == FT2
+type FT1 function(int ... a);
+type FT2 function(int[] a);
+
+//@type FT1Ret == FT2Ret
```

Comment:
These are completely different.  Think of all the arguments of a function as a tuple. The type of the argument list of FT1Ret is `[int...]`; the type of the argument list of FT2Ret if `[int[]]`. (So `function (int, int)` has argument tuple type `[int,int]`.)

Reply:
fixed. (I am not sure if this is a bug but new test cases also passed without any changes to the rest of the code.)